configman/config_manager.py

```diff
@@ -461,7 +461,10 @@ def write_conf(self, config_file_type, opener, skip_keys=None):
                 an_option = option_defs[a_key]
                 if ((not a_key.startswith('admin'))
                     and isinstance(an_option, Option)
-                    and an_option.secret
+                    and (
+                        an_option.secret
+                        or 'password' in an_option.name.lower()
+                    )
```

Comment:
I mean, if you don't want `password_hint` or `min_password_length` to get caught up in the automatic redaction your only option is to set `expose_secrets=False` which doesn't seem nice.